### PR TITLE
OPS-111: Misc. tidy ups

### PR DESCRIPTION
**Pull request by rb-vatnz**

### CODE REVIEW

File: docs/aerodromes/Class-C/nzaa.md [hunk 1]
```diff
@@ -92,13 +92,13 @@ All aircraft pushing from the International apron shall be given a tail facing i
...
 !!! info
-    The start instruction is only utilised in the real-world to ensure tha the jet wash doesn't interfere with the aircraft stands on the opposite side of the taxiway. As this isn't an issue within the Simulator, it is up to Controller discretion.
+    The start instruction is only utilised in the real-world to ensure that the jet wash doesn't interfere with the aircraft stands on the opposite side of the taxiway. As this isn't an issue within the Simulator, it is up to Controller discretion.
 
 ### Taxiing 
 
 As the runway utilises high-speed exits, the Controller shall avoid using taxiway `A` when there is a potential for conflict between the taxiway and any vacating traffic. 
 
-When Runway 23L is in operation aircraft will vacate at either `A6` or `A8`, and `A7`, `A5` or `A3` when 05R is in use.
+When Runway 23L is in operation aircraft will vacate at either `A4`, `A6` or `A8`, and `A7`, `A5` or `A3` when 05R is in use.
```
Comment (rb-vatnz) [inline docs/aerodromes/Class-C/nzaa.md:101]: `A4` added as its listed on Minimum Runway Occupancy chart for Light aircraft.

File: docs/aerodromes/Class-C/nzaa.md [hunk 2]
```diff
@@ -137,7 +137,7 @@ For Domestic aircraft that are departing to the North or Northeast, the Controll
 
 | Runway | Procedure  | Allowed A/C Categories | Is divergent?                  | Notes                                                                                                                       |
 | ------ | ---------- | ---------------------- | ------------------------------ | --------------------------------------------------------------------------------------------------------------------------- |
-| 05R    | `BROOK #Q` | Cat A to C             | Yes - >30° turn passing `A005` | Available to Cat C Jets for noise-abatement. AA TMR approval required.                                                      |
+| 05R    | `BROOK #Q` | Cat A to C             | Yes - >30° turn passing `A005` | Available to Cat C Jets for noise-abatement. AA TMA approval required.                                                      |
```
Comment (rb-vatnz) [inline docs/aerodromes/Class-C/nzaa.md:140]: Changed to `TMA` just to match terminology being used elsewhere in this section

File: docs/enroute/BAY.md [hunk 3]
```diff
@@ -65,7 +65,7 @@ BAY shall issue STARs to aircraft bound for NZNP with coordination, and may desc
 
 ### AA TMA
 
-BAY shall issue STAR clearances to aircraft bound for any NZAA, NZAR or NZWP without coordination, provided it is for the nominated duty runway and approach type as stated in the ATIS. A request for use of a non-nominated approach (such as an RNP) requires agreement from both the TMA and ADC Controller.
+BAY shall issue STAR clearances to aircraft bound for any of NZAA, NZAR or NZWP without coordination, provided it is for the nominated duty runway and approach type as stated in the ATIS. A request for use of a non-nominated approach (such as an RNP) requires agreement from both the TMA and ADC Controller.
```
Comment (dirtyformal) [inline docs/enroute/BAY.md]: Might pay to change this to "..aircraft bound for NZAA, NZAR or NZWP.. - dropping the "any of".

Comment (rb-vatnz) [reply docs/enroute/BAY.md]: Ah yes that makes it much better!

**Pull request merged**